core/trino-main/src/main/java/io/trino/execution/SqlTaskManager.java

```diff
@@ -14,9 +14,11 @@
 package io.trino.execution;
```

Comment:
please shorten the commit message (~72 chars too avoid wrapping) and add more info as commit description: https://cbea.ms/git-commit/